code/modules/power/lighting.dm

```diff
@@ -82,6 +82,7 @@
 	layer = EFFECTS_LAYER_UNDER_1
 	plane = PLANE_NOSHADOW_ABOVE
 	text = ""
+	var/rigged = 0 // allows some older maps to compile
```

Comment:
nope, fix the maps instead - this won't do anything and it's an extremely bad code smell

Reply:
maps fixed